agir/people/admin/panels.py

```diff
@@ -99,6 +100,25 @@ class TagListFilter(AutocompleteFilter):
     title = "Tags"
 
 
+class AnimateMoreThanOneGroup(admin.SimpleListFilter):
+    title = "Cette personne annime plus d'un groupe d'action"
+    parameter_name = "Person who animate more than one group"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("animate_more_than_one_group", "Anime plus d'un groupe"),
+            ("", ""),
+        )
+
+    def queryset(self, request, queryset):
+        result = []
+        if self.value() == "animate_more_than_one_group":
+            for p in queryset:
```

Comment:
Une petite boucle sur un millier de lignes ? Le serveur va adorer ^^

Petite piste avec les annotations et les aggrégations pour effectuer toutes ces opérations en base de donnée :
```suggestion
            return queryset.annotate(animated_groups=Count("memberships", filter=Q(membership__membership_type__gte=Membership.MEMBERSHIP_TYPE_MANAGER)).filter(animated_groups__gt=1)
```

À adapter et préciser probablement, plus de doc ici : https://docs.djangoproject.com/fr/3.2/topics/db/aggregation/